config/perf_configs/bm_perf_target.yml

```diff
@@ -1,16 +1,17 @@
 moist: "equil"
 apply_limiter: true
-z_elem: 25
+z_elem: 11
```

Comment:
Why is this changing?

Reply:
I think it's because this job failed: https://buildkite.com/clima/climaatmos-ci/builds/22496#0194af81-d116-4343-92f6-9b753e5763aa, which is likely related to the climatimestepper update.

You changed the wrong config file though I think?

Reply:
Ah, ok. I think it might be better to just increase the mem request

Reply:
Using low resolution is not really meaningful for performance monitoring jobs, so I think it'd be better to remove it instead of lowering the resolution.

Reply:
That's fine with me. I prefer to make it soft fail though so we will remember to come back to fix this.

Reply:
> Ah, ok. I think it might be better to just increase the mem request

The memory is GPU memory, so we would need to request a different type of GPU, which would increase the queue time significantly.

Reply:
I changed it to a soft fail

Reply:
Sounds good. I think there is something wrong, though. The fieldvector is being allocated here, and it's definitely not that large, so I don't know why we're OOMing. Maybe the GPU is not freeing memory as it should be.